Assignment2/u120210.rdf

```diff
@@ -0,0 +1,9 @@
+@base <http://www.oeg-upm.net/ontologies/people/ >
+@prefix vocab: <http://www.ontologies.org/ontologies/ejercicio2>
+:Class01 vocab:includes :Sensor029 ;
+vocab:includes: Computer101.
+:Sensor029 vocab:hasMeasurement :Measurement8401.
+:Computer01 vocab:hasOwner :User10A.
+:Measurement8401 vocab:hasTemperature "29" ;
```

Comment:
As this is an integer value, the object of the triple should be datatyped with xsd:int